docker/start.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
```

Comment:
Can we add `set -e -o pipefail` here? If ocw-to-hugo fails for example, we would want `docker-compose up` to fail too